add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: Directly embedding user input within SQL queries can lead to SQL injection vulnerabilities. This is a critical security issue.

**Recommendations**: Use prepared statements to safely handle user input. Here’s an example using `mysqli`:

```php
$stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
$stmt->bind_param("ssi", $title, $author, $year);
$stmt->execute();
$stmt->close();
```

This approach ensures that user input is properly escaped and avoids SQL injection risks.